release/Magarena/scripts/Nezumi_Shortfang.groovy

```diff
@@ -0,0 +1,44 @@
+def action = {
+    final MagicGame game, final MagicEvent event ->
+        final int hand = event.getPlayer().getHandSize();
```

Comment:
function body should be at the same indentation level as the parameters in the previous line